### PR TITLE
Wrap get_sad bench in black_box.

### PR DESCRIPTION
**Pull request by tdaede**

### CODE REVIEW

File: benches/me.rs [hunk 1]
```diff
@@ -45,7 +45,7 @@ fn bench_get_sad(b: &mut Bencher, bs: &BlockSize) {
   let plane_ref = rec_plane.slice(&po);
 
   b.iter(|| {
-      let _ = me::get_sad(&plane_org, &plane_ref, bsw, bsh);
+      let _ = black_box(me::get_sad(&plane_org, &plane_ref, bsw, bsh));
```
Comment (lu-zero) [inline benches/me.rs]: I was thinking about:

``` rust
black_box(me::get_sad)(&plane_org, &plane_ref, bsw, bsh);
```

Comment (tdaede) [reply benches/me.rs]: I can test that to see if it measures any different.

Comment (tdaede) [reply benches/me.rs]: So doing that is totally ineffective at black boxing, it turns out. You have to use a value from the function with black_box.

Comment (lu-zero) [reply benches/me.rs]: Thank you for testing :)

**Pull request merged**